xml/docker_building_images.xml

```diff
@@ -192,25 +174,28 @@ RUN zypper ref -s
 RUN zypper -n in vim</screen>
   </sect2>
 
-  <sect2 xml:id="sec.docker.sle_images.customizing_the_images.sles11sp3">
-   <title>Creating a Custom &slea;&nbsp;11 SP3 Image</title>
+<sect2 xml:id="sec.docker.sle_images.customizing_the_images.sles15">
+   <title>Creating a Custom &slea;&nbsp;15 Image</title>
```

Comment:
use the \&productname; here and on line +181?

Reply:
This would make the title quite long, I fear.... At first glance, the guide uses a lot of abbreviated product names anyway (unfortunately). Maybe I will have a closer look at this after this PR is merged and closed, because replacing the abbreviations with proper product names might require to rephrase some section titles.

Reply:
@taroth21 Ah --- sorry, I meant \&productnumber; instead of the plain "15".